client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-expanded-content/backup-storage.tsx

```diff
@@ -88,7 +90,35 @@ export default function BackupStorage( { site }: Props ) {
 		strong: <strong></strong>,
 	};
 
-	const isBackupEnabled = hasBackupError ? false : site.has_backup;
+	const hasBackup = site.has_backup;
+	const isBackupEnabled = ! hasBackupError && hasBackup;
+
+	const siteUrlWithMultiSiteSupport = urlToSlug( site.url );
+	// If the backup has an error, we want to redirect the user to the backup page
+	const link = hasBackupError ? `/backup/${ siteUrlWithMultiSiteSupport }` : '';
+
+	const { isLicenseSelected, handleAddLicenseAction } = useDashboardAddRemoveLicense(
+		site.blog_id,
+		DASHBOARD_LICENSE_TYPES.BACKUP
+	);
+
+	const addBackupText = isLicenseSelected ? (
+		<span className="site-expanded-content__license-selected">
+			<Gridicon icon="checkmark" size={ 16 } />
```

Comment:
Used Gridicon here since it is being used in the columns and looks a bit different.

Gridicon 


<img width="341" alt="Screenshot 2023-03-27 at 3 24 11 PM" src="https://user-images.githubusercontent.com/10586875/227908183-11e4b36f-26b8-4387-a684-9e3b782a5527.png">


Icon

<img width="346" alt="Screenshot 2023-03-27 at 3 22 54 PM" src="https://user-images.githubusercontent.com/10586875/227908166-3c86ecb8-d215-4398-920d-e67c5a80298e.png">